### DISCUSSION TITLE
How to remove tables?

### DISCUSSION THREAD

**Discussion by agruenberg** [Budibase/budibase#8050] [Help] [answered]
First of all I want to say a big thank you all for your effort to establish such a great product!

I am struggling a little bit with the MySQL Connection: I renamed a table and since them the old one is still in the table list and cannot be removed. Any suggestion how to remove it? I need a way to do this without to fetch all tables again, because I would loose all formula columns.

Kind!

**Comment by poirazis** [comment]
Hey @agruenberg 
Use the "Fetch Tables" button and it should refresh your table list and update to reflect any changes made in the underlying MySQL 

<img width="764" alt="Screenshot 2022-09-30 at 1 48 23 PM" src="https://user-images.githubusercontent.com/80512170/193254214-47b82468-aa34-4b7a-a999-84c8d9036ebd.png">

**Reply by agruenberg** [reply]
Thank you @poirazis but in this case I would loose all formula columns, wouldn't I? 
Fetching only the concerning table didn't help.

**Reply by poirazis** [reply]
Unfortunately yes. But this is being worked on as I understand. To circumvent this as I use MySQL too, I added the needed formula columns as virtual in MySql.

**Reply by agruenberg** [reply]
Ah, I understand. Thank you very much!

**Reply by gmentzik** [reply]
Do you know if this loosing of formulas is fixed? In latest version fetch tables behavior changed! What do you mean by virtual columns?

**Reply by ConorWebb96** [reply, marked as answer]
Hey @gmentzik,

I've hopefully answered your question here; https://github.com/Budibase/budibase/discussions/11232.
This was only added recently, so this wasn't available on earlier versions of Budibase, I believe it was added from 2.8.2 onwards.

I hope this helps.

**Reply by melohagan** [reply]
As Conor mentioned in the linked answer, formula columns are no longer removed when you fetch tables.
